griptape/drivers/image_generation_model/bedrock_stable_diffusion_image_generation_model_driver.py

```diff
@@ -107,30 +107,21 @@ def _request_parameters(
         text_prompts = [{"text": prompt, "weight": 1.0} for prompt in prompts]
         text_prompts += [{"text": negative_prompt, "weight": -1.0} for negative_prompt in negative_prompts]
 
-        request = {"text_prompts": text_prompts, "cfg_scale": self.cfg_scale}
-
-        if self.style_preset is not None:
-            request["style_preset"] = self.style_preset
-
-        if self.clip_guidance_preset is not None:
-            request["clip_guidance_preset"] = self.clip_guidance_preset
-
-        if self.sampler is not None:
-            request["sampler"] = self.sampler
-
-        if image is not None:
-            request["init_image"] = image.base64
-            request["width"] = image.width
-            request["height"] = image.height
-        else:
-            request["width"] = width
-            request["height"] = height
-
-        if self.steps is not None:
-            request["steps"] = self.steps
-
-        if seed is not None:
-            request["seed"] = seed
+        request = {
+            "text_prompts": text_prompts,
+            "cfg_scale": self.cfg_scale,
+            **({"style_preset": self.style_preset} if self.style_preset is not None else {}),
```

Comment:
This approach seems unnatural, are we sure we want a rule that requires this?

Reply:
We do use it in other places but maybe wrongly so. What do you think about the updated approach?